### DISCUSSION TITLE
Download Documentation for Offline Usage

### DISCUSSION THREAD

**Discussion by 4l1fe** [litestar-org/litestar#3000] [Q&A] [answered]
Hey, i've tried to find the static documentation files an Artifacs, but failed.

Is there a place where i could download it from?

**Comment by euri10** [comment]
you mean something like in https://devdocs.io/ ?

**Reply by 4l1fe** [reply]
I mean static files of the https://docs.litestar.dev/latest/index.html

**Comment by provinzkraut** [comment]
Currently this does not exist but we could offer a download e.g. as a PDF.

**Reply by 4l1fe** [reply, marked as answer]
I've just downloaded recent artifacts from here https://github.com/litestar-org/litestar-docs-preview/actions/runs/7594205746 and seems to be it is what i'm looking for.

I went before to the same builds, but that time artifacts were expired and deleted by Github.
